osu.Game/Tests/Visual/TestCaseEditor.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using System.Collections.Generic;
+using osu.Framework.Allocation;
+using osu.Game.Rulesets;
+using osu.Game.Screens.Edit;
+using osu.Game.Screens.Edit.Screens;
+using osu.Game.Tests.Beatmaps;
+
+namespace osu.Game.Tests.Visual
+{
+    public abstract class TestCaseEditor : OsuTestCase

```

Comment:
maybe call this `EditorTestCase` in line with `OsuTestCase`?

Reply:
Fine with `TestCaseEditor : EditorTestCase` ?

Reply:
yeah, just staying in line with the other abstract test case implementations

Reply:
cool